sympy/physics/mechanics/body.py

```diff
@@ -155,6 +161,72 @@ def z(self):
         """The basis Vector for the Body, in the z direction. """
         return self.frame.z
 
+    def kinetic_energy(self, frame):
+        """Kinetic energy of the body.
+
+        Explanation
+        ===========
+
+        The kinetic energy, T, of a Body(particle), P, is given by
+
+        'T = 1/2 m v^2'
+
+        where m is the mass of body P, and v is the velocity of the
+        particle in the supplied ReferenceFrame.
+
+        The kinetic energy, T, of a Body(rigid body), B, is given by
+
+        'T = 1/2 (I omega^2 + m v^2)'
```

Comment:
This is only true for a 2D planar motion.

Reply:
See equation (4) here: https://ocw.mit.edu/courses/aeronautics-and-astronautics/16-07-dynamics-fall-2009/lecture-notes/MIT16_07F09_Lec27.pdf

Reply:
So what should it be?

Reply:
Equation (4)

Reply:
@moorepants Equation (6) is the simplified form of equation (4) and is being used in docstring.